src/runtime/thread_pool.cc

```diff
@@ -394,9 +397,43 @@ int TVMBackendParallelLaunch(
     FTVMParallelLambda flambda,
     void* cdata,
     int num_task) {
+#ifndef USE_OMP
   int res = tvm::runtime::ThreadPool::ThreadLocal()->Launch(
       flambda, cdata, num_task, 1);
   return res;
+#else
+  const char *val = getenv("TVM_NUM_THREADS");
+  int num_workers;
+  if (val == nullptr) {
+    val = getenv("OMP_NUM_THREADS");
+  }
+  if (val != nullptr) {
+    num_workers = atoi(val);
+  } else {
+#if defined(_M_X64) || defined(__x86_64__)
+  // Half to not count hyper threading.
+    num_workers = std::thread::hardware_concurrency() / 2;
+#else
+    num_workers = std::thread::hardware_concurrency();
+#endif
+  }
+  num_workers = std::max(num_workers, 1);
```

Comment:
This part is a reuse of https://github.com/dmlc/tvm/blob/master/src/runtime/threading_backend.cc#L218